src/rez/tests/test_packages_order.py

```diff
@@ -334,6 +335,148 @@ def test_pod(self):
         }))
 
 
+class TestPyPAPackageOrder(_BaseTestPackagesOrder):
+    """Test case for the PyPAPackageOrder class"""
+
+    def test_reorder(self):
+        """Validate package ordering with a PyPAPackageOrder"""
+        # First test that we can sort packages with no prerelease allowance,
+        # so we expect full versions on top.
+        orderer = PyPAPackageOrder()
+        expected = [
+            # Full versions and post releases first.
+            '1.0.1',
+            '1.0.0post1',
+            '1.0.0+local',
+            '1.0.0',
+
+            # Followed by higher version prerelease versions, sorted according to pypa
```

Comment:
Ideally these would be filtered.  I assume there's no way for an orderer to do that?

Reply:
No, unfortunately.  That's possible via package_filters though, which we could explore.  Keep in mind that if you filter them out, they are actually not available for people to even specify explicitly.  By ordering them to the back, we can still do a `rez env package-1.0.0b2` or whatever.